products/kubernetes-csi-node-driver-registrar.md

```diff
@@ -0,0 +1,105 @@
+---
+title: kubernetes-csi-node-driver-registrar
+category: app
+tags: kubernetes
+permalink: /kubernetes-csi-node-driver-registrar
+alternate_urls:
+-   /kubernetes-csi-node-driver-registrar
+versionCommand: |-
+  csi-node-driver-registrar --version
+releasePolicyLink: https://kubernetes-csi.github.io/docs/node-driver-registrar.html#supported-versions
+changelogTemplate: https://github.com/kubernetes-csi/node-driver-registrar/releases/tag/v__LATEST__
+releaseDateColumn: true
+eolColumn: Support
+
+auto:
+  methods:
+  -   git: https://github.com/kubernetes-csi/node-driver-registrar
+
+identifiers:
+-   repology: node-driver-registrar
```

Comment:
https://repology.org/project/node-driver-registrar/versions says Unknown project.

```suggestion
```

Reply:
I couldn't find it in repology.org what do you suggest me to do? If this is mandatory, I should go and create a entry for this project in repology first.

Reply:
no, the suggested change is to just drop the entry.

Reply:
thanks, removed!